tests/kafkatest/services/kafka/kafka.py

```diff
@@ -89,11 +91,38 @@ def start(self):
                 topic_cfg["topic"] = topic
                 self.create_topic(topic_cfg)
 
+    def set_protocol_and_port(self, node):
```

Comment:
I think that makes sense @benstopford. @rajinisivaram, is there a reason why you made the change referred by @benstopford above?